.github/workflows/release-demo.yaml

```
@@ -54,8 +54,8 @@ jobs:
         id: netlify
         if: steps.check.outputs.changed == 'true'
         env:
-          NETLIFY_SITE_ID: ${{ secrets.NETLIFY_SITE_ID }}
-          NETLIFY_AUTH_TOKEN: ${{ secrets.NETLIFY_AUTH_TOKEN }}
+          NETLIFY_ARCHIVE_SITE_ID: ${{ secrets.NETLIFY_ARCHIVE_SITE_ID }}
+          NETLIFY_ARCHIVE_AUTH_TOKEN: ${{ secrets.NETLIFY_ARCHIVE_AUTH_TOKEN }}
```

These are used by netlify and are predefined, we just need to update the naming of our secrets.
```suggestion
          NETLIFY_SITE_ID: ${{ secrets.NETLIFY_ARCHIVE_SITE_ID }}
          NETLIFY_AUTH_TOKEN: ${{ secrets.NETLIFY_ARCHIVE_AUTH_TOKEN }}
```